testsuite/features/support/remote_node.rb

```diff
@@ -167,47 +167,47 @@ def wait_while_process_running(process)
     end
   end
 
-  # Copies a local file to a remote node.
+  # Copies a file from the test runner (aka controller) into the remote node.
   #
-  # @param local_file [String] The path to the file to copy.
-  # @param remote_file [String] The path in the destination.
+  # @param test_runner_file [String] The path to the file to copy.
+  # @param remote_node_file [String] The path in the destination.
   # @return [Integer] The exit code.
-  def inject(local_file, remote_file)
+  def inject(test_runner_file, remote_node_file)
     if @has_mgrctl
       tmp_folder, _code = run_local('mktemp -d')
-      tmp_file = File.join(tmp_folder.strip, File.basename(local_file))
-      code, _remote = scp(local_file, tmp_file)
-      if code.zero?
-        _out, code = run_local("mgrctl cp --user #{user} #{tmp_file} server:#{remote_file}")
```

Comment:
Does this really need to be removed?

Reply:
This line?
`        _out, code = run_local("mgrctl cp --user #{user} #{tmp_file} server:#{remote_file}")
`

Reply:
Yes. But I am not sure. I only saw that you kept it in the `extract()` method below.

Reply:
I moved it to this line:
```
      if success
        _out, code = run_local("mgrctl cp --user #{user} #{tmp_file} server:#{remote_node_file}")
```

Reply:
Ah, yes. Sorry my bad. I missed that. All good then.